packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -163,66 +174,45 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
   }
 
   updated() {
+    const { _isMobileVersion } = this;
     this._isLTR = window.getComputedStyle(this).direction === 'ltr';
     this._activeTabIndex = parseInt(this._activeTab, 10);
 
-    this._tabItems.map((tab, index) => {
+    this._tabItems.forEach((tab, index) => {
       (tab as DDSTab).selected = index === this._activeTabIndex;
       (tab as DDSTab).setIndex(index);
-      const navLink = this.shadowRoot!.querySelectorAll(
-        `.${prefix}--tabs__nav-link`
-      )[index];
-      const navText = navLink!.querySelector('div p');
-      if (navText!.scrollHeight > navText!.clientHeight) {
-        const label = (tab as DDSTab).getAttribute('label');
-        if (label) {
-          navLink!.setAttribute('aria-label', label);
-          navLink!.setAttribute('hasTooltip', label);
+
+      // Attach accordion behavior on mobile.
+      if (_isMobileVersion) {
+        tab.addEventListener('click', (e) => this._handleClick(index, e));
+      }
```

Comment:
We should wrap the update stuff here to only happen if the value of `this._isMobileVersion` has changed to minimize blocking the main thread